apps/fe-admin/src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { NavigationProps } from '../../types';
+import clsx from 'clsx';
+import styles from './Navigation.module.scss';
+import { useLocation, NavLink } from 'react-router-dom';
+
+const Navigation: React.FC<NavigationProps> = ({ links }) => {
```

Comment:
no need to move `NavigationProps` outside the component, remove it from types folder
if you keep your props outside the component, it will be a mess

Reply:
not fixed